runtime/compiler/il/J9DataTypes.hpp

```diff
@@ -56,6 +56,18 @@ enum TR_BCDSignCode
    num_bcd_sign_codes = 4
    };
 
+enum TR_arrayTypeCode
```

Comment:
These aren't already defined some where?

Reply:
I can't find it in the JIT. @DanHeidinga is there enum for array type code defined in the VM?

Reply:
Surprisingly no.  We use the code as an index into the J9JavaVM's array of array classes:
https://github.com/eclipse/openj9/blob/25d479f31ac710fc17922b8ce0419770e47d5548/runtime/vm/BytecodeInterpreter.hpp#L7455